libraries/AP_HAL_Linux/RCInput_RPI.cpp

```diff
@@ -286,6 +286,7 @@ void *RCInput_RPI::map_peripheral(uint32_t base, uint32_t len)
 
     if (fd < 0) {
         printf("Failed to open /dev/mem: %m\n");
+        printf("Make sure that CONFIG_STRICT_DEVMEM is disabled if running OS64\n");
```

Comment:
it's not specific to 64b. CONFIG_STRICT_DEVMEM may be enabled regardless of being 64 or 32 bits. It's just that it's being enabled by default by more recent distros

Reply:
@lucasdemarchi,

Thanks for the review. Could you suggest better wording maybe?

Reply:
applied.